engine/src/main/java/org/terasology/engine/rendering/assets/mesh/VertexDefinition.java

```diff
@@ -0,0 +1,24 @@
+// Copyright 2021 The Terasology Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+package org.terasology.engine.rendering.assets.mesh;
+
+import org.terasology.engine.rendering.assets.mesh.resource.VertexAttribute;
+import org.terasology.engine.rendering.assets.mesh.resource.VertexResource;
+
+/**
+ *
+ */
```

Comment:
This might be obvious for someone who knows their way around graphics programming, but for me Javadoc would be really helpful. 

If this is a very common pattern, maybe we can give cross links (to something like OpenGL specs or docs or whatever).